Helper/ClientManager.php

```diff
@@ -54,26 +50,38 @@ public function __construct(
         $this->moduleList = $moduleList;
     }
 
-    public function initClient($storeId = null)
+    public function getClient($storeId = null)
```

Comment:
I see `getClient` used in areas of the code that I believe need to be updated, e.g. Mail/Transport.

Reply:
@aingram it was originally updated in Mail/Transpor at my fork, but when I was back porting to mainline repo I forgot to include Mail/Transpor changes. Below you will find CR Changes in 717c07d. Thanks for good catch!

Reply:
Does this file, line 82, need to be updated? That method, `getSettings`, is used by `apiValidate` which is used by `Block/System/Config/Button/Check.php`. `apiValidate` is also used by `isValid` in this file, which is used by `Block/System/Config/Api/FieldRenderer.php`

Does `Helper/Api.php` (line 125) need to be updated? It seems to be used in
* `Plugin/GroupIntercept.php`: It calls `$apiHelper->isValid()` which calls `apiValidate` which calls this method.
* `VerifiedEmails.php`: `$emails = $this->apiHelper->client->getVerifiedSenders();`